packages/core/admin/ee/server/controllers/admin.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+// eslint-disable-next-line node/no-extraneous-require
+const ee = require('@strapi/strapi/lib/utils/ee');
+const { env } = require('@strapi/utils');
+const { getService } = require('../../../server/utils');
+
+module.exports = {
+  async licenseLimitInformation() {
+    const permittedSeats = ee.seats;
+
+    let shouldNotify = false;
+    let licenseLimitStatus = null;
+    let enforcementUserCount;
+
+    const currentActiveUserCount = await getService('user').getCurrentActiveUserCount();
+
+    const eeDisabledUsers = await getService('seat-enforcement').getDisabledUserList();
+
+    if (eeDisabledUsers) {
+      enforcementUserCount = currentActiveUserCount + eeDisabledUsers.length;
```

Comment:
shouldn't this be the number of active users minus the number of disabled ones?

Reply:
nope as we want to know the current active + the one we disabled automatically to tell them they have 12 instead of 10 for ex. 12 disabled + 10 active

Reply:
ooh, I see 🆗